src/main/java/net/visma/autopay/http/signature/SignatureContext.java

```diff
@@ -182,6 +189,28 @@ public Builder targetUri(String targetUri) {
             return this;
         }
 
+        /**
+         * Sets target URI
+         * <p>
+         * Method offers compatibility with {@code HttpServletRequest} methods.
+         * {@code targetUri} is not the same as servlet's {@code requestUri}.
+         *
+         * @param requestUrl  request URL
+         * @param queryString query string
+         * @return This builder
+         */
+        public Builder targetUri(StringBuffer requestUrl, String queryString) {
```

Comment:
Could you add example call? Here, and in README.md: Full example -> Creating signature -> Signature context.

Also, be prepared to write release notes. See https://github.com/Visma-AutoPay/http-signatures/releases

Reply:
example call added, release notes prepared